test/extended/builds/pipeline.go

```diff
@@ -155,6 +156,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			// start the build
 			go func() {
 				defer g.GinkgoRecover()
+				defer wg.Done()
```

Comment:
if i understand these changes, then move shouldn't be necessarily since the timeout is going to ensure we don't get hung waiting for a "done" anyway.  Not that i'm against this change, it's cleaner, just testing my understanding of the changes.